# p2planetest2.py가 요구하면 half_sphere.py가 값을 준단



---
float64[] view1
float64[] view2
float64[] view3
float64[] view4